README.md

```diff
@@ -64,6 +66,22 @@ Options:
   --help                          Show this message and exit.
 ```
 
+## API server
```

Comment:
We should add some information about the possible operations allowed by this API, in a general sense. It could also be useful to include a YAML in the project to document them.